CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to
 ## [Unreleased]
 
 - Documenting support for python 3.13. (#86)
+- Ignore imported rules and filters when building the rule registry. (#85)
```

Comment:
```suggestion
- Ignore imported rules and filters when building the rule registry. (#87)
```

Reply:
Let's refer to the current PR, instead of the bug report. This allows e.g. PRs to partially solve bug reports, and therefore the changelog to be more precise :)